tutorials/tmva/RBatchGenerator_NumPy.py

```diff
@@ -27,10 +27,22 @@
 )
 
 # Loop through training set
-for i, b in enumerate(ds_train):
-    print(f"Training batch {i} => {b.shape}")
+i = 1
+while True:
+    try:
+        b = next(ds_train)
+        print(f"Training batch {i} => {b.shape}")
+        i+=1
+    except StopIteration:
+        break
```

Comment:
This is really ugly to show in a tutorial, I would just revert this change

Reply:
Reverted in the next PR